components/kyma-environment-broker/internal/process/provisioning/input/input.go

```diff
@@ -67,8 +67,8 @@ func (r *RuntimeInput) SetRuntimeLabels(instanceID, subAccountID string) interna
 	defer r.mutex.Unlock("SetRuntimeLabels")
 
 	r.input.RuntimeInput.Labels = &gqlschema.Labels{
-		brokerKeyPrefix + "instance_id":   []string{instanceID},
-		globalKeyPrefix + "subaccount_id": []string{subAccountID},
+		brokerKeyPrefix + "instance_id":   instanceID,
+		globalKeyPrefix + "subaccount_id": subAccountID,
```

Comment:
Does input request to provisioner shouldn't be changed also? See graphQL input [builder](https://github.com/kyma-incubator/compass/blob/master/components/kyma-environment-broker/internal/provisioner/graphqlizer.go#L56)